build/openresty/openssl/openssl_repositories.bzl

```diff
@@ -20,6 +20,5 @@ def openssl_repositories():
         strip_prefix = "openssl-" + version,
         urls = [
             "https://github.com/openssl/openssl/releases/download/openssl-" + version + "/openssl-" + version + ".tar.gz",
-            "https://openssl.org/source/old/3.1/openssl-" + version + ".tar.gz",
```

Comment:
i think this was added during the openssl.org side refactor. the site is now recovered. so inside of removing it here, we can add a `"https://github.com/openssl/openssl/releases/download/openssl-" + version + "/openssl-" + version + ".tar.gz"`